web/server/codechecker_server/api/authentication.py

```diff
@@ -245,11 +245,62 @@ def createLink(self, provider):
                 code_verifier=pkce_verifier
             )
 
-        self.__insertOAuthSession(state=state,
-                                  code_verifier=pkce_verifier,
-                                  provider=provider)
+        self.insertOAuthSession(state=state,
+                                code_verifier=pkce_verifier,
+                                provider=provider)
         return url
 
+    @timeit
+    def validateOAuthTokenSession(self, access_token):
```

Comment:
don't add it as a public function. Make it a server internal function. You can also move this db query to the test code.